tests/test_clientsecrets.py

```diff
@@ -222,12 +224,18 @@ def test_validate_error(self):
             except clientsecrets.InvalidClientSecretsError as e:
                 self.assertTrue(str(e).startswith(match))
 
-    def test_load_by_filename(self):
+    def test_load_by_filename_missing_file(self):
+        exc_caught = None
         try:
             clientsecrets._loadfile(NONEXISTENT_FILE)
-            self.fail('should fail to load a missing client_secrets file.')
-        except clientsecrets.InvalidClientSecretsError as e:
-            self.assertTrue(str(e).startswith('File'))
+        except clientsecrets.InvalidClientSecretsError as exc:
+            exc_caught = exc
+
+        expected_args = ('Error opening file',
```

Comment:
Let's not make the exact text of error messages part of our definition of system correctness. Would it be enough just to match the filename and error constant?